src/modules/notes/controllers/updateNote.js

```diff
@@ -1,11 +1,22 @@
 import { HttpStatusCode } from '#src/constants';
 import { CustomAPIError } from '#errors/custom-error';
 import { PROVIDE_VALUES } from '../constants/index.js';
+import { schemas } from '../schemas.js';
 
 export const updateNote = (req, res) => {
-	const noParameters = !Object.keys(req.body).length;
+	const { body } = req;
+	const noParameters = !Object.keys(body).length;
 	if (noParameters) {
 		throw new CustomAPIError(HttpStatusCode.BAD_REQUEST, PROVIDE_VALUES);
 	}
+
+	const result = schemas.noteUpdate.validate(body);
+	const { error } = result;
+	const valid = error == null;
```

Comment:
!

Reply:
By the way, we can describe validation inside middlewear, as you wish

Reply:
thanks! moved validation to middleware